android/src/main/java/io/rownd/android/Rownd.kt

```diff
@@ -248,14 +253,72 @@ class RowndClient constructor(
         }
     }
 
+    private fun runRememberedRequestSignIn() {
+        rememberedRequestSignIn?.let {
+            it()
+            rememberedRequestSignIn = null
+        }
+    }
+
+    private fun isAppConfigLoadingWithCallback(callback: () -> (Unit)): Boolean {
+        val scope = CoroutineScope(Dispatchers.IO)
+        val isLoading = state.value.appConfig.isLoading
+        if (isLoading) {
+            scope.launch {
+                store.stateAsStateFlow().collect() {
```

Comment:
I don't know if this is important (Kotiln noob), but should this just be `store.sateAsStateFlow().collect {` not `.collect()`?  That keeps it consistent with other places like https://github.com/rownd/android/pull/42/files#diff-b94b4258af013d86803e0118bd7f7d3354d84b329ea8029bddc3c3b6ddf49444R164

Reply:
Yup that works. Nice catch!